static/usr/lib/core/lockout-not-enabled.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -eu
+
+! [ -f /etc/writable/faillock.enabled ]
```

Comment:
This now needs to be `/etc/writable/account-locked.enabled` after the latest tweaks in https://github.com/snapcore/snapd/pull/12278